src/System.Security.Cryptography.X509Certificates/tests/project.json

```diff
@@ -13,6 +13,7 @@
     "System.Security.Cryptography.Algorithms": "4.2.1-beta-24311-02",
     "System.Security.Cryptography.Cng": "4.2.1-beta-24311-02",
     "System.Security.Cryptography.Encoding": "4.0.1-beta-24311-02",
+    "System.Security.Cryptography.OpenSsl": "4.0.1-beta-24311-02",
```

Comment:
Why does OpenSsl need to be added? I don't see any OpenSsl types being talked about.